torch/_dynamo/variables/nn_module.py

```diff
@@ -216,7 +216,8 @@ def record_nn_module_stack():
                 # unroll Sequential()
                 assert not kwargs
                 (arg,) = args
-                for child_name, submod in mod.named_children():
+                # TODO: Use named_children when it supports remove_duplicate.
```

Comment:
this message is confusing; it sounds like you want remove_duplicate=True, but you actually want remove_duplicate=False, right?

Reply:
Yes, let me make it more clear.